core/field_checkbox.js

```diff
@@ -220,6 +220,6 @@ FieldCheckbox.prototype.convertValueToBool_ = function(value) {
   }
 };
 
-Blockly.fieldRegistry.register('field_checkbox', FieldCheckbox);
+register('field_checkbox', FieldCheckbox);
```

Comment:
I'm pretty sure that `fieldRegistry.register` would be more readable here.

Reply:
Done